### DISCUSSION TITLE
Issues with hsl function in webpack

### DISCUSSION THREAD

**Discussion by HaveSpacesuit** [iTwin/iTwinUI#628] [General]
After installing @itwin/itwinui-react 1.36.0 (which is using @itwin/itwinui-css 0.54.1), I get a build error related to these lines in the style/theme.scss file:

```scss
$iui-color-background-border--light-hc: hsl(0 0% 0% / 0.3);

$iui-color-background-border--dark-hc: hsl(0 0% 100% / 0.34);
```

The error I get:
![image](https://user-images.githubusercontent.com/15036346/166979556-87dfb116-0775-417c-89c3-55bdb267d49a.png)

viewStatus.scss has the following statement: 
```scss
@import '~@itwin/itwinui-css/scss/index';
```

We build with webpack, and use style-loader 0.23.1, css-loader 3.6.0, and sass-loader 7.1.0. I've tried updating the sass-loader to version 10, which is the highest version which will support our version of webpack, but it is the same issue with a different wrapper:

![image](https://user-images.githubusercontent.com/15036346/166987386-67341cf8-f38e-4d10-8d45-bcce20059a7a.png)

I can tweak the file on local runs to use the hsla function such as `hsla(0, 0%, 0%, .3)` and remove problems.

**Comment by mayank99** [comment]
Thanks for creating this discussion.

> viewStatus.scss has the following statement:
> 
> ```css-scss
> @import '~@itwin/itwinui-css/scss/index';
> ```

Is there any reason you are importing our `scss` rather than the compiled css we provide?

**Reply by HaveSpacesuit** [reply]
In that file, we are using it for a mixin like `@include iui-text(small)`. Is there a better way to do that?

**Reply by mayank99** [reply]
Firstly, I would suggest trying to use our `<Text>` or `<Small>` react components as they are considered stable (1.0+) because itwinui-css is 0.X and we might make breaking changes.

But if you really need to use that mixin, then I think importing from `@itwin/itwinui-css/scss/text/mixins` should fix the error.

**Reply by HaveSpacesuit** [reply]
Thanks Mayank! Changing the import location did not help, but using the `<Text>` component worked great.
